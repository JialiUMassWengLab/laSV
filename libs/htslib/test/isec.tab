1	3062915	3062915
1	3106154	3106154
1	3157410	3162006
1	3177144	3177144
4	3212016	3212016
