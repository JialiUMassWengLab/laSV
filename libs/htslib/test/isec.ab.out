1	3157410	GA	G
1	3162006	GAA	G
1	3177144	GT	G
